### PR TITLE
Add NAIS Guides section

### PR DESCRIPTION
**Pull request by Starefossen**
* [x] Getting started with NAIS

### CODE REVIEW

File: docs/guides/basics.md [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+
+### Prerequisites
+
+Before you get started, you need to make sure you have the following:
+
+- [x] You have a GitHub account
+- [x] You have a GCP account
+- [x] You have a working nais device
+- [x] [You have a NAIS team](../basics/teams.md#creating-a-new-team)
+- [x] [You have a NAIS API key](../basics/teams.md#access-to-api-keys)
```
Comment (jksolbakken) [inline docs/guides/basics.md:24]: Skip `you have` in he bullet points.

File: docs/guides/basics.md [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+gh repo create <my-repo> --add-readme --clone
+```
+
+For your new repository add the following secret:
+
+```bash
+gh secret set NAIS_DEPLOY_APIKEY --app actions
+```
+
+Paste in your NAIS API key as the value when asked.
```
Comment (jksolbakken) [inline docs/guides/basics.md:91]: Where do you obtain this APIKEY?

**Review by jksolbakken — CHANGES_REQUESTED** [review]
Excellent start!

File: docs/guides/basics.md (hunk 2 above)
Comment (Starefossen) [reply docs/guides/basics.md:91]: Listed under prerequsite

**Pull request merged**